hack/generator/pkg/astmodel/interface_implementer.go

```diff
@@ -46,31 +46,31 @@ func (i InterfaceImplementer) AsDeclarations(
 
 	var result []ast.Decl
 
-	// First interfaces must be ordered by name for deterministic output
-	var ifaceNames []TypeName
-	for ifaceName := range i.interfaces {
-		ifaceNames = append(ifaceNames, ifaceName)
+	// interfaces must be ordered by name for deterministic output
+	// (We sort them directly to skip future lookups)
+	var interfaces []*InterfaceImplementation
+	for _, intf := range i.interfaces {
+		interfaces = append(interfaces, intf)
 	}
 
-	sort.Slice(ifaceNames, func(i int, j int) bool {
-		return ifaceNames[i].name < ifaceNames[j].name
+	sort.Slice(interfaces, func(i int, j int) bool {
+		return interfaces[i].Name().name < interfaces[j].Name().Name()
 	})
 
-	for _, ifaceName := range ifaceNames {
-		iface := i.interfaces[ifaceName]
+	for _, intf := range interfaces {
```

Comment:
minor: Is `intf` a more clear abbreviation for interface than `iface`? I did write it the other way so am clearly biased but I think `iface` is better =)

Reply:
😁 My understanding is abbreviated terms where later letters are omitted are easier for readers to decipher, which is why I prefer `intf` over `iface` (don't recall ever seeing `iface` before). 

But, no biggie, I'll change it back.